projects/fgia_accv2022_1st/config/README.md

```diff
@@ -0,0 +1,56 @@
+# This is the official implementation of ViT-L pre-training for the 1st place of FGIA ACCV2022
+
```

Comment:
Better to indicate the version for pretraining

Reply:
We can use the collect_env to list important information, like, mmcv, mmengine, mmcls, and pytorch, cuda, torchvision